stack.yaml

```diff
@@ -2,7 +2,7 @@ extra-package-dbs: []
 packages:
 - '.'
 
-resolver: lts-18.13
+resolver: lts-18.19
```

Comment:
fyi the latest Stackage lts-18 version is lts-18.27 and discord-haskell works perfectly fine with it

Reply:
looks like it's lts-18.28, which I'll try now